app/models/test/client_request.rb

```diff
@@ -0,0 +1,23 @@
+module Test
```

Comment:
it is for test env only, yes, but it needs to be a real ActiveRecord class. I don't care where it lives.